client/signup/steps/domains/index.jsx

```diff
@@ -86,7 +86,6 @@ class DomainsStep extends React.Component {
 	};
 
 	getDefaultState = () => ( {
-		previousStepSectionName: this.props.stepSectionName,
 		suggestion: null,
```

Comment:
`suggestion` is not used either. This component doesn't have any state. It used to manage trademark claim notices (added in #32552) but that code was (incompletely) removed in #35861.

Reply:
Indeed, cleaned up in 7c42937dd0219cfcdeb6bed8296c7f183b7e498d